doc/index.rst

```diff
@@ -6,40 +6,55 @@
 lsst.ts.MTDomeTrajectory
 ########################
 
-ts_MTDomeTrajectory contains the `MTDomeTrajectory` CSC and support code,
-including a simple `MockDome` CSC for unit testing.
+.. image:: https://img.shields.io/badge/SAL\ Interface-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/MTDomeTrajectory.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_MTDomeTrajectory
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_MTDomeTrajectory
 
-.. _lsst.ts.MTDomeTrajectory-using:
+Overview
+========
 
-Using lsst.ts.MTDomeTrajectory
-==============================
+MTDomeTrajectory commands the Simonyi Survey Telescope dome to follow the telescope.
+It does this by reading telescope position from the MTMount CSC and issuing commands to the Dome CSC.
 
-You can setup and build this package using eups and sconsUtils.
-After setting up the package you can build it and run unit tests by typing ``scons``.
-Building it constructs a version.py file and runs unit tests.
+.. _lsst.ts.MTDomeTrajectory-user_guide:
 
-To run the `MTDomeTrajectory` CSC type ``runMTDomeTrajectory.py``
+User Guide
+==========
 
-.. _lsst.ts.MTDomeTrajectory-contributing:
+Start the MTDomeTrajectory CSC as follows:
 
-Contributing
-============
+.. prompt:: bash
 
-``lsst.ts.MTDomeTrajectory`` is developed at https://github.com/lsst-ts/ts_MTDomeTrajectory.
-You can find Jira issues for this module using `labels=ts_MTDomeTrajectory <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20labels%20%20%3D%20ts_MTDomeTrajectory>`_.
+    run_mtdometrajectory.py
 
-.. If there are topics related to developing this module (rather than using it), link to this from a toctree placed here.
+Stop the CSC by sending it to the OFFLINE state.
 
-.. .. toctree::
-..    :maxdepth: 1
+To make dome track the telescope send the MTDomeTrajectory CSC to the ENABLED state.
 
-.. _lsst.ts.MTDomeTrajectory-pyapi:
+To stop the dome from tracking the telescope (e.g. if you want to send the dome to some specific position) send the MTDomeTrajectory CSC to the DISABLED state (or any state other than ENABLED).
 
-Python API reference
-====================
+MTDomeTrajectory supports multiple algorithms for following the telescope. The algorithm is specified in the :ref:`configuration <lsst.ts.MTDomeTrajectory-configuration>`.
 
-.. automodapi:: lsst.ts.MTDomeTrajectory
-   :no-main-docstr:
+MTDomeTrajectory does not support simulation. However you can run the MTDome CSC in simulation mode.
```

Comment:
I think the point here is not that MTDomeTrajectory does not "support" simulation, it's that it just doesn't need it. Maybe re-structure this a little bit?
"MTDomeTrajectory does not include any simulation capabilities. However, the CSC can be fully exercised when used in conjunction with MTDome CSC running in simulation mode."
It would probably be helpful to link the MTDome documentation page here...

Reply:
Good suggestion. How about the following?
```
MTDomeTrajectory can be fully exercised without hardware by using it with the MTMount and Dome CSCs running in simulation mode.
Thus MTDomeTrajectory does not have, or need, a simulation mode.
```

Reply:
I have added links to the MTMount and Dome docs -- great suggestion. I hope Dome will be renamed to MTDome, which will break that link, but the docs will need a minor update at that point in any case.